src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -31,7 +46,18 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
      */
     public function getPosition();
 
+    /**
+     * Defines the configuration key used to reference the provider
+     * in the firewall configuration.
+     * 
+     * @return string
+     */
     public function getKey();
 
+    /**
+     * Adds custom security configuration options.
+     * 
+     * @param NodeDefinition $builder
+     */
```

Comment:
I agree, it could be formulated in a different way though; 
```php
/**
 * Adds additional security configuration for this specific authenticator.
 *
 * @param NodeDefinition $builder
 */
public function addConfiguration(NodeDefinition $builder);
```